src/applications/mhv-secure-messaging/tests/e2e/utils/constants.js

```diff
@@ -219,7 +222,7 @@ export const Locators = {
   },
   CHECKBOX: {
     CL_ALL: '[data-testid="contact-list-select-all-teams"]',
-    CL_SINGLE: '[data-testid="contact-list-select-team"]',
+    CL_SINGLE: '[data-testid*="contact-list-select-team"]',
```

Comment:
For my known knowledge sake, why is this "*" placed here?

Reply:
'*' is for partial value - it will select all elements with `data-testid = "contact-list-select-team-whatever_will_be_here"`

Reply:
Nice, thank you!